liquibase-dist/src/main/archive/liquibase.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Maybe this file should be called something like `liquibase-env.sh`?  First time I saw it I thought it was a wrapper to liquibase command.

Reply:
👍🏻